src/seedu/addressbook/logic/Logic.java

```diff
@@ -85,7 +85,9 @@ public CommandResult execute(String userCommandText) throws Exception {
     private CommandResult execute(Command command) throws Exception {
```

Comment:
Since the behaviour of the execute method has changed, you should update its header comments to reflect the changes.